nevergrad/optimization/optimizerlib.py

```diff
@@ -1668,3 +1717,36 @@ def __init__(self, parametrization: IntOrParameter, budget: Optional[int] = None
                     self.optims = [CMA(self.parametrization, budget, num_workers)]
             else:
                 self.optims = [NGO(self.parametrization, budget, num_workers)]
+
+
+@registry.register
+class MetaModel(base.Optimizer):
```

Comment:
Are you planning on adding more MetaModels? If this is a specific case, a more specific name may be better for later compatibility

Reply:
Other metamodels are clearly a thing, but the quadratic one is by far the most widely used. So I would say it makes sense to consider this as "the" metamodel by default.